Packs/CiscoStealthwatch/Integrations/CiscoStealthwatch/cisco_stealthwatch.py

```diff
@@ -0,0 +1,613 @@
+        outputs_key_field='id',
+        raw_response=response,
+        outputs=outputs,
+        readable_output=table)
+
+
+def cisco_stealthwatch_query_flows_results_command(client: Client, tenant_id: str,
+                                                   search_id: str) -> CommandResults:
+    """Retrieve the results for a query flow by search id
+
+    Args:
+        client (Client): Cisco Stealthwatch Client
+        tenant_id (str): The id of the tenant the search was performed on
+        search_id (str): The id of the search
+
+    Returns:
+        CommandResults: Raw response, outputs and readable outputs
+    """
+    response = client.get_flow_search_results(tenant_id, search_id)
+    outputs = []
+    for data in dict_safe_get(response, ['data', 'flows']):
+        outputs.append(data)
+    headers = ['id', 'tenantId', 'flowCollectorId', 'protocol', 'serviceId', 'statistics', 'peer',
+               'subject']
+    table = tableToMarkdown('Query Flows Results Information:', outputs, headers=headers,
+                            removeNull=True)
+    return CommandResults(
+        outputs_prefix='CiscoStealthwatch.FlowResults',
+        outputs_key_field=['id', 'tenantId'],
+        raw_response=response,
+        outputs=outputs,
+        readable_output=table
+    )
+
+
+def cisco_stealthwatch_list_tags_command(client: Client, tenant_id: str) -> CommandResults:
+    """List tags (called host groups on the Stealthwatch API) based on tenant id
+
+    Args:
+        client (Client): Cisco Stealthwatch Client
+        tenant_id (str): The id of the tenant to list its tags (tenant is a domain on the API)
+
+    Returns:
+        CommandResults: Raw response, outputs and readable outputs
+    """
+    response = client.list_tags(tenant_id)
+    outputs = []
+    for tag in response.get('data', []):
+        outputs.append(tag)
+
+    outputs = sorted(outputs, key=lambda x: x.get('id'))
+
+    table = tableToMarkdown(f'Tags (for tenant_id: {tenant_id}):', outputs,
+                            headers=['displayName', 'id'], removeNull=True)
+
+    return CommandResults(
+        outputs_prefix='CiscoStealthwatch.Tag',
+        outputs_key_field='id',
+        raw_response=response,
+        outputs=outputs,
+        readable_output=table
+    )
+
+
+def cisco_stealthwatch_get_tag_command(client: Client, tenant_id: str, tag_id: str)\
+        -> CommandResults:
+    """Get a single tag (called host group on the Stealthwatch API) information
+
+    Args:
+        client (Client): Cisco Stealthwatch Client
+        tenant_id (str): The id of the tenant to get its tag information
+        tag_id (str): The id of the tag to retrieve its information
+
+    Returns:
+        CommandResults: Raw response, outputs and readable outputs
+    """
+    response = client.get_tag(tenant_id, tag_id)
+    outputs = response.get('data', {})
+
+    table = tableToMarkdown(f'Tag (tenant_id: {tenant_id}, tag_id: {tag_id}):', outputs,
```

Comment:
Same here, I personally think that brackets don't look very good in a title, maybe something like this? 
`f'Tag {tag_id} with tenant id: {tenant_id} results:'`

Please fix in all commands human readable 🙏